tools/run_tests/xds_k8s_test_driver/framework/infrastructure/traffic_director.py

```diff
@@ -290,6 +293,33 @@ def delete_forwarding_rule(self, force=False):
         self.compute.delete_forwarding_rule(name)
         self.forwarding_rule = None
 
+    def create_firewall_rule(self,
+                             force=False,
```

Comment:
I don't think `force` is ever used in `create_` methods. This is mostly to force `delete_` methods to force resource deletion, even if this specific instance of TrafficDirectorManager doesn't manage them.

Reply:
Oops. I misunderstood your last comment. The `force` argument is removed.